install/lib/class.FroxlorInstall.php

```diff
@@ -487,6 +487,14 @@ private function _doSettings(&$db)
 		if ($this->_data['webserver'] == "apache24") {
 			$this->_updateSetting($upd_stmt, 'apache2', 'system', 'webserver');
 			$this->_updateSetting($upd_stmt, '1', 'system', 'apache24');
+			
+			// on redhat systems set different defaults
+			if(file_exists('/etc/redhat-release')) {
```

Comment:
too specific, also these are all __settings__ you can adjust, unnecessary here

Reply:
Its as unspecific as i could make it (it will work on all the redhat-based distro's).
It would also give an works-out-of-the-box experience similar to that given to debian/ubuntu distro's.

Reply:
see new commit